shared/protocol/src/main/java/io/pravega/shared/NameUtils.java

```diff
@@ -341,6 +351,29 @@ public static String getSystemJournalFileName(int containerId, long epoch, long
         return String.format(SYSJOURNAL_NAME_FORMAT, epoch, containerId, currentFileIndex);
     }
 
+
+    /**
+     * Gets file name of SystemJournal snapshot for given container instance.
+     * @param containerId The Id of the Container.
+     * @param epoch Epoch of the container instance.
+     * @param currentSnapshotIndex Current index for journal file.
+     * @return File name of SystemJournal for given container instance
+     */
+    public static String getSystemJournalSnapshotFileName(int containerId, long epoch, long currentSnapshotIndex) {
+        return String.format(SYSJOURNAL_SNAPSHOT_NAME_FORMAT, epoch, containerId, currentSnapshotIndex);
```

Comment:
We probably need some unit tests for these.

Reply:
added